build_tools/github_actions/lint_llvm_commit.sh

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 The StableHLO Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+print_usage() {
+  echo "Usage: $0 [-f] <path/to/llvm/version>"
+  echo "    -f           Auto-fix whitespace issues."
+}
+
+FORMAT_MODE='validate'
+while getopts 'f' flag; do
+  case "${flag}" in
+    f) FORMAT_MODE="fix" ;;
+    *) print_usage
+       exit 1 ;;
+  esac
+done
+shift $(( OPTIND - 1 ))
+
+if [[ $# -ne 1 ]] ; then
+  print_usage
+  exit 1
+fi
+
+LLVM_VERSION_DIR=$1
```

Comment:
Quotes around `"$1"`, also is this argument a dir or is this  the path to `llvm_version.txt`, if so consider `LLVM_VERSION_TXT`?